keyboards/handwired/replicazeron/promicro/info.json

```
@@ -0,0 +1,11 @@
+{
+    "usb": {
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["C6", "D4", "D7", "E6", "F7"],
+        "rows": ["B1", "B3", "B2", "B6", "B5", "B4"]
+    },
+    "processor": "atmega32u4",
+    "bootloader": "qmk-dfu"
```

dfu isn't correct for a pro micro, by default.  Also.
```suggestion
    "development_board": "promicro"
```